### PR TITLE
Java Solution for The Party And Sweets

### PR DESCRIPTION
**Pull request by brad182**

### CODE REVIEW

File: solutions/cf-1158A.mdx [hunk 1]
```diff
@@ -97,5 +97,59 @@ int main(){
...
+
+		for (int x = 0; x < n; x++) {
+			b[x] = io.nextInt();
+			res += b[x];
+		}
+
+		for (int x = 0; x < m; x++) {
+			g[x] = io.nextInt();
+		}
+
```
Comment (nathangong) [inline solutions/cf-1158A.mdx:126]: comments from cpp code
```suggestion

	// Initial lower bound on answer
```

File: solutions/cf-1158A.mdx [hunk 2]
```diff
@@ -97,5 +97,59 @@ int main(){
...
+		res *= m;
+
+		Arrays.sort(b);
+		Arrays.sort(g);
+
+		if (b[n - 1] > g[0]) {
```
Comment (nathangong) [inline solutions/cf-1158A.mdx:132]:
```suggestion
		// If it's impossible to satisfy the constraints
		if (b[n - 1] > g[0]) {
```

**Comment by xyzyzl** [comment]
bruh just commit these small things

**Comment by brad182** [comment]
yep forgot to copy over the comments

**Review by maggie-j-liu — APPROVED** [review]

**Review by dongliu0426 — APPROVED** [review]

**Pull request merged**